infra/modules/service/workflow_orchestrator_role.tf

```diff
@@ -0,0 +1,110 @@
+#--------------------------------
+# Scheduler Workflow Manager Role
+#--------------------------------
+# This role and policy are used by the Step Functions state machine that manages the scheduled jobs workflow.
+
+resource "aws_iam_role" "workflow_orchestrator" {
+  name                = "${var.service_name}-workflow-orchestrator"
+  managed_policy_arns = [aws_iam_policy.workflow_orchestrator.arn]
+  assume_role_policy  = data.aws_iam_policy_document.workflow_orchestrator_assume_role.json
+}
+
+data "aws_iam_policy_document" "workflow_orchestrator_assume_role" {
+  statement {
+    sid     = "ECSTasksAssumeRole"
```

Comment:
Looks like this name was copy pasted, rename or maybe we don't even need a SID since there's only one statement

Reply:
Updated! https://github.com/navapbc/platform-test/pull/130/commits/367badc4c79286c9215692073b227ac4234d6446